src/PKSim.CLI.Core/Batch/Compound.cs

```diff
@@ -2,13 +2,13 @@
 
 namespace PKSim.Core.Batch
 {
-   public class PkaType
```

Comment:
I have marked all those classes as internal for now. There is another entry to remove them once we are ready to convert all batch to snapshot (this cannot be done for now because we have no way to only export parameters that have actually changed. This will be possible once Model Documentation is done and we have ValueOrigin parameters)